include/rounding.h

```diff
@@ -26,6 +26,8 @@ std::pair <NT, NT> rounding_min_ellipsoid(Polytope &P , const std::pair<Point,NT
     typedef typename Polytope::MT 	MT;
     typedef typename Polytope::VT 	VT;
     typedef typename Parameters::RNGType RNGType;
+    typedef Eigen::Matrix<NT, Eigen::Dynamic,1> Coeff;
```

Comment:
This has to be given as a template `VT`

Reply:
Actually it is redundant. We can use the VT type from 2 line above.